R/utilities-writing-report.R

```diff
@@ -231,10 +231,26 @@ renderWordReport <- function(fileName, intro = NULL, createWordReport = FALSE, w
         grepl(pattern = "figure", x = anchorName)
       )
     )
+    bookmarkLines <- c(
+      "",
+      "```{=openxml}",
+      paste0(
+        "<w:p><w:r>",
+        # Page break
+        '<w:br w:type="page"/>',
+        "</w:r><w:r>",
+        # Bookmark that uses anchore name (e.g. figure-1-1)
+        '<w:bookmarkStart w:id="', anchorName, '" w:name="', anchorName, '"/>',
+        '<w:bookmarkEnd w:id="', anchorName, '"/>',
+        "</w:r></w:p>"
+        ),
+      "```",
+      ""
+      )
```

Comment:
@pchelle Can you add the link to the webpage with this hint as a comment?

Reply:
In case, also adding the link as an answer here
https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.wordprocessing.bookmarkstart?view=openxml-2.8.1#remarks